src/containers/Admin/index.tsx

```diff
@@ -24,22 +25,24 @@ import StopPlaceSearch from './StopPlaceSearch'
 import './styles.scss'
 
 const AdminPage = ({ history }: Props): JSX.Element => {
-    const position = useMemo(() => getPositionFromUrl(), [])
+    const position = usePosition()
```

Comment:
Klarar me å droppe `usePosition` og heller hente `position` frå settings? Og så lar me heller `useSettings` finne position i URL for gamle tavler?

Reply:
Jeg skal ta en titt på det! Det blir nok ryddigere